lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -10,24 +11,40 @@ class PokemonOverviewPage extends StatelessWidget {
     Key? key,
   }) : super(key: key);
 
-  final List<Pokemon> pokemons;
+  final Async<List<Pokemon>> pokemons;
 
   @override
   Widget build(BuildContext context) {
+    void showNoPokemonAvailable(errorMessage) {
```

Comment:
tho this is allowed, please don't make it a practice to create a function inside a function unless it's super needed 🙂 

have this outside then make it private instead 🙂